ui/pages/home/home.component.js

```diff
@@ -831,6 +845,7 @@ export default class Home extends PureComponent {
             ? this.renderOnboardingPopover()
             : null}
           {showMultiRpcEditModal && <MultiRpcEditModal />}
+          {displayUpdateModal && <UpdateModal />}
```

Comment:
Any idea what happens when multiple models want to be displayed at the same time?